tasks/build.py

```diff
@@ -346,21 +346,60 @@ def download_pr(repo_name, branch_name, pr, arch_job_dir):
     # - 'curl' diff for pull request
     # - 'git apply' diff file
     git_clone_cmd = ' '.join(['git clone', f'https://github.com/{repo_name}', arch_job_dir])
-    clone_output, clone_error, clone_exit_code = run_cmd(git_clone_cmd, "Clone repo", arch_job_dir)
+    print(f'cloning with command {git_clone_cmd}')
```

Comment:
This prints to `stdout`. Should it maybe go to a log file?

Reply:
Absolutely, I've changed this and other occurrences to use `log()`.